<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Clean Image Folder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : #161219;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cleanOriginal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>320</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border :2px solid '#BA1F33' ;
	border-radius: 15px ;
	font-size : 15px ;
	color: 'white' ;

	margin : 0px 20px ;
}

QPushButton:hover{
	background: '#BA1F33' ; 
	color: '#161219';
}</string>
    </property>
    <property name="text">
     <string>Choose an image from the folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="smallLogo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../logo/small_logo_rb.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanDone">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>190</y>
      <width>130</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border :4px solid '#BA1F33' ;
	border-radius: 15px ;
	font-size : 20px ;
	color: 'white' ;

	margin : 10px 20px ;
}

QPushButton:hover{
	background: '#BA1F33' ; 
	color: '#161219';
}</string>
    </property>
    <property name="text">
     <string>DONE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
